test/src/models/form_group_test.dart

```diff
@@ -78,7 +78,7 @@ void main() {
           value: 'hello',
           validators: [Validators.required],
         ),
-        'email': FormControl(validators: [
+        'email': FormControl<String>(validators: [
```

Comment:
This will not allow creating a control with dynamic type as:
```dart
final control = FormControl(
   value: "I'm a String control",
   validators: [Validators.email], // exception is thrown
);
```
So I suggest leaving the Email validator as **dynamic**